sentry-core/src/main/java/io/sentry/core/transport/RetryingThreadPoolExecutor.java

```diff
@@ -28,33 +32,33 @@
  * <p>This class is not public because it is used solely in {@link AsyncConnection}.
  */
 final class RetryingThreadPoolExecutor extends ScheduledThreadPoolExecutor {
-  private final int maxRetries;
   private final int maxQueueSize;
   private final AtomicInteger currentlyRunning;
-  private final IBackOffIntervalStrategy backOffIntervalStrategy;
+
+  private static final int HTTP_TOO_MANY_REQUESTS = 429;
+  static final long HTTP_RETRY_AFTER_DEFAULT_DELAY = 60000; // default 60s
```

Comment:
Maybe indicate the unit. Ms or Mills or Milliseconds
```suggestion
  static final long HTTP_RETRY_AFTER_DEFAULT_DELAY_MS = 60000; // default 60s
```

Reply:
makes sense